File: 01-programming/java/snippets/operators_math.java
import java.util.Scanner;

public class operators_math {
  public static void main(String[] agrs) {

    Scanner input = new Scanner(System.in);

    System.out.println("Please enter a number: ");
    String a = input.nextLine();

    System.out.println("Please enter another number: ");
    String b = input.nextLine();

    int num = Integer.parseInt(a);
    int num2 = Integer.parseInt(b);

    System.out.println(num + " + " + num2 + " = " + (num + num2));
    System.out.println(num + " - " + num2 + " = " + (num - num2));
    System.out.println(num + " * " + num2 + " = " + (num * num2));
    System.out.println(num + " / " + num2 + " = " + (num / num2));

    input.close();
  }
}

File: 01-programming/cpp/snippets/operators_math.cpp
#include <iostream>

int main() {
  int num, num2;
  std::cout << "Please enter a number: ";
  std::cin >> num;
  std::cout << "Please enter another number: ";
  std::cin >> num2;

  std::cout << num << " + " << num2 << " = " << num + num2 << "\n";
  std::cout << num << " - " << num2 << " = " << num - num2 << "\n";
  std::cout << num << " * " << num2 << " = " << num * num2 << "\n";
  std::cout << num << " / " << num2 << " = " << num / num2 << "\n";
  return 0;
}
